INFO:root:discriminator/aggregate/transform_aggregate/fully_connected/biases:0 ((300,)): 300 params
INFO:root:discriminator/aggregate/fully_connected/weights:0 ((300, 3)): 900 params
INFO:root:discriminator/aggregate/fully_connected/biases:0 ((3,)): 3 params
INFO:nli-debug-cli.py:Total Trainable Discriminator Parameters: 16019703
INFO:tensorflow:Restoring parameters from models/snli/esim_1/acl//batch_dsearch_reg_v12e/esim_1_15
INFO:tensorflow:Restoring parameters from models/snli/esim_1/acl//batch_dsearch_reg_v12e/esim_1_15
  0%|          | 0/8 [00:00<?, ?it/s] 12%|█▎        | 1/8 [00:01<00:08,  1.20s/it] 25%|██▌       | 2/8 [00:02<00:06,  1.08s/it] 38%|███▊      | 3/8 [00:03<00:05,  1.07s/it] 50%|█████     | 4/8 [00:04<00:04,  1.01s/it] 62%|██████▎   | 5/8 [00:05<00:03,  1.02s/it] 75%|███████▌  | 6/8 [00:06<00:02,  1.00s/it] 88%|████████▊ | 7/8 [00:06<00:00,  1.00it/s]100%|██████████| 8/8 [00:07<00:00,  1.05it/s]
INFO:nli-debug-cli.py:Number of examples: 972
INFO:nli-debug-cli.py:Accuracy: 0.4784
INFO:nli-debug-cli.py:(S1 contradicts S2) XOR (S2 contradicts S1): 0
INFO:nli-debug-cli.py:(S1 contradicts S2): 0
./bin/nli-debug-cli.py:323: RuntimeWarning: invalid value encountered in long_scalars
  logger.info('(S1 contradicts S2) AND NOT(S2 contradicts S1): {0} ({1:.4f})'.format(c.sum(), c.sum() / b.sum()))
INFO:nli-debug-cli.py:(S1 contradicts S2) AND NOT(S2 contradicts S1): 0 (nan)
INFO:nli-debug-cli.py:(S1 entailment S2): 188
INFO:nli-debug-cli.py:(S1 entailment S2) AND (S2 contradicts S1): 0 (0.0000)
INFO:nli-debug-cli.py:(S1 neutral S2): 784
INFO:nli-debug-cli.py:(S1 neutral S2) AND (S2 contradicts S1): 0 (0.0000)
INFO:nli-debug-cli.py:(True): 1944 [972 + 972]
INFO:nli-debug-cli.py:(True) AND NOT(S1 entails S1): 0 [0 + 0] (0.0000)
